platform/android/java/lib/src/org/godotengine/godot/io/file/DataAccess.kt

```diff
@@ -170,14 +170,17 @@ internal abstract class DataAccess(private val filePath: String) {
 		}
 	}
 
-	fun write(buffer: ByteBuffer) {
+	fun write(buffer: ByteBuffer): Boolean {
 		try {
+			val totalBytes = buffer.position()
```

Comment:
According to https://developer.android.com/reference/java/nio/channels/WritableByteChannel#write(java.nio.ByteBuffer), `remaining()`may be the one to use instead.